uptane-standard.md

```diff
@@ -72,196 +72,462 @@ This document describes a framework for securing automotive software update syst
+outsource the production of ECUs. Tier-1 suppliers directly serve the
+manufacturers. Tier-2 suppliers are those that receive outsourced work
+from Tier-1 suppliers.
 
-## Design Requirements
+*Telematics Systems*: Electronic system that can remotely monitor a vehicle
+or its individual components.
 
-# Detailed Design of Uptane
+*Vehicle Version Manifest*: A compliation of manifests from all ECUs on
+a vehicle. It serves as a master list of all programs currretly
+running on alll ECUs in the vehcile.
 
-Uptane is a secure software update framework for automobiles. We do not specify implementation details. Instead, we describe the components necessary for a compliant implementation, and leave it up to individual implementors to make their own technological choices within those requirements.
+
+## **2.3. Uptane Role Terminology**
+
+These terms are defined in greater detail in Section 5.
+
+*Delegations*: Designating the responsibility of signing metadata to another entity. 
+
+*Roles*: The roles mechanism of Uptane allows
+the system to distribute signing responsibilities so compromise
+of one key does not impact other parts of the system.
+
+  * *Root Role*: Distributes and revokes public keys used to 
+  verify the root, timestamp, release, and targets role metadata.
+
+  * *Snapshot Role*: Indicates which images the repository has released at the
+  same time.
+  
+  * *Targets Role*: Holda the metadata used to verify the image 
+  
+  * *Timestamp Role*: Indicates if there are any new metadata or image 
+  on the repository
+
+
+## **2.4 Acronyms and Abbreviations**
+*CAN Bus*: Controller Area Network bus standard.
+
+*ECUs*: Electronic Control Units, the computing units on vehicle.
+
+*LIN Bus*: Local Interconnect Bus
+
+*SOTA*: Software Over-the-Air
+
+*VIN*: Vehicle identification Number
+
+# **3. Rationale for and Scope of Uptane Standards**
+This Standards document clarifies the essential components and best practices 
+for the secure design implentation and deployment of Uptane by OEMs and suppiers.
+These practices contribute to compromise resilience, or the ability
+to minimize the extent of the threat posed by any given attack.
+
+## **3.1. Why Uptane requires standards**
+A standards document that can guide the safe design, integration and deployment of
+Uptane in cars is needed at this time because:
+
+* The number of connected units on the average vehicle continues to grow, with 
+mainstream cars now containing up to [10 million lines](https://www.usatoday.com/story/tech/columnist/2016/06/28/your-average-car-lot-more-code-driven-than-you-think/86437052/)
+of code.
+
+* The [expanded use of software over-the-air](https://www.consumerreports.org/automotive-technology/automakers-embrace-over-the-air-updates-can-we-trust-digital-car-repair/)
+strategies creates new attack surfaces for malicious parties.
+
+* Legacy update strategies, such as SSL/TLS or GPG/RSA, are not feasible for use
```

Comment:
This might not even be what belongs here. I got a little confused because in the sample standard that Ira was nice enough to pass along to me---because I had no idea what this was supposed to address--was providing a rationale for the standard and not the technology. So, I was sort of splitting the difference here in making a case for both Uptane and the new standard. As a result it may be too muddled, but I figured I'd let someone who knows better than me resolve what information this section needed.